.github/workflows/build.yml

```diff
@@ -14,23 +14,23 @@ jobs:
       fail-fast: false
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
Why the version changes as part of this PR? I think it's ok to include but should be separated out to another commit for a total of 2 commits.

Reply:
Partially, yes. I had to upgrade to v3 in codeql in previous commit. This just aligns the versioning for build as well as I missed it.